org.OpenT2T.Sample.SuperPopular.Thermostat/org.OpenT2T.Sample.SuperPopular.Thermostat.xml

```diff
@@ -10,10 +10,20 @@
     <method name="turnOff"/>
     
     <!-- Get the current ambient temperature -->
-    <method name="getCurrentTemperature">
+    <method name="getAmbientTemperature">
+      <!-- Temperature scale:  'f' for Fahrenheit. 'c' for celsius -->
+      <arg name="scale" type="s" direction="in" />
       <arg name="temp" type="d" direction="out" />
     </method>
 
+    <!-- Set the current ambient temperature -->
+    <method name="setTargetTemperature">
```

Comment:
There are already methods for this below, they are called setHeatingSetpoint and setCoolingSetpoint. Is there a reason they don't work for Nest?